pycartool/io/montage.py

```diff
@@ -7,28 +7,26 @@
 from mne.channels import make_dig_montage
 import numpy as np
 
-
-def read_xyz(filename, kind=''):
+def read_xyz(filename):
     """Reads and convert xyz positions to a mne montage type
 
     Parameters
     ----------
-    filename : str or file-like
-        The Electrodes coordinates (.xyz) file to read.
-
+    filename : str
+        The filename of the xyz file.
     Returns
     -------
-    montage : mne.channels.DigMontage
```

Comment:
I think make_dig_montage returns an instance of DigMontage.
https://mne.tools/stable/generated/mne.channels.make_dig_montage.html?highlight=make_dig_montage